addon/components/ember-flatpickr.js

```diff
@@ -37,10 +37,12 @@ import { getOwner } from '@ember/application';
  * @uses Flatpickr
  */
 export default class EmberFlatpickr extends Component {
-  date = null;
-  flatpickrRef = null;
```

Comment:
If we are using `this.flatpickrRef` for things, we probably need to set it somewhere. Where is it being set?

Reply:
There is no need to set it because the `field` already contains the `_flatpickr` property when `flatpickr()` is called. Which is why I moved `flatpickrRef` to a getter property.